src/renderer/components/spinner/spinner.scss

```diff
@@ -35,9 +35,7 @@
   }
 
   &.centerHorizontal {
-    position: absolute;
-    left: 50%;
-    margin-left: calc(var(--spinner-size) / -2);
+    margin-left: 50%;
```

Comment:
I don't see any other code applying `centralHorizontal` to a `Spinner`, but is this a generally useful change? If I apply `centralHorizontal` to a `Spinner` in another context will it have an expected effect?

It's also hard to see the effect in your videos. The spinner moved lower and to the right a bit? Still seems to be over the extension cluster menu.